tardis/io/config_internal.py

```diff
@@ -37,15 +35,11 @@ def get_data_dir():
             get_config_dir(), "tardis_internal_config.yml"
         )
         logging.critical(
-            "\n{line_stars}\n\nTARDIS will download different kinds of data (e.g. atomic) to its data directory {default_data_dir}\n\n"
-            "TARDIS DATA DIRECTORY not specified in {config_file}:\n\n"
-            "ASSUMING DEFAULT DATA DIRECTORY {default_data_dir}\n "
-            "YOU CAN CHANGE THIS AT ANY TIME IN {config_file} \n\n"
-            "{line_stars} \n\n".format(
-                line_stars="*" * 80,
-                config_file=config_fpath,
-                default_data_dir=DEFAULT_DATA_DIR,
-            )
+            f"\n{"*" * 80}\n\nTARDIS will download different kinds of data (e.g. atomic) to its data directory {DEFAULT_DATA_DIR}\n\n"
+            f"TARDIS DATA DIRECTORY not specified in {config_fpath}:\n\n"
+            f"ASSUMING DEFAULT DATA DIRECTORY {DEFAULT_DATA_DIR}\n "
+            f"YOU CAN CHANGE THIS AT ANY TIME IN {config_fpath} \n\n"
+            f"{"*" * 80} \n\n"
```

Comment:
This is invalid syntax. Please run `python setup.py develop` or use linting to check for syntax errors.